### PR TITLE
Nav Search Additional Feature

### PR DESCRIPTION
**Pull request by Favy-source**
The search box has been fixed @konfirmed

### CODE REVIEW

File: theme.config.tsx [hunk 1]
```diff
@@ -24,6 +24,11 @@ const config: DocsThemeConfig = {
   feedback: {
     content: null
   },
+
+  search: {
+    placeholder: 'search'
```
Comment (konfirmed) [inline theme.config.tsx:29]: Capitalize the 'S'earch

**Review by konfirmed — COMMENTED** [review]
@Favy-source thanks

**Comment by Favy-source** [comment]
I'm merging this since it includes all updated changes @konfirmed

**Review by Favy-source — COMMENTED** [review]
Merging

**Pull request merged**